server/api/impl/configurator/configurator.go

```diff
@@ -27,15 +24,17 @@ type config struct {
 
 	lock             *sync.RWMutex
 	mm               *pluginapi.Client
+	awsClient        *aws.Client
 	mattermostConfig *model.Config
 }
 
-func NewConfigurator(mattermost *pluginapi.Client, buildConfig *api.BuildConfig, botUserID string) api.Configurator {
+func NewConfigurator(mattermost *pluginapi.Client, awsClient *aws.Client, buildConfig *api.BuildConfig, botUserID string) api.Configurator {
```

Comment:
0/5 style with the `Service`-oriented code structure, manage and cache `p.aws` at the `plugin` level, can be refactored later